plugins/inputs/bind/bind_test.go

```diff
@@ -0,0 +1,200 @@
+package bind
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBindJsonStats(t *testing.T) {
+	ts := httptest.NewServer(http.FileServer(http.Dir("testdata")))
+	defer ts.Close()
+
+	b := Bind{
+		Urls: []string{ts.URL + "/json/v1"},
```

Comment:
Ok, I've enabled gather_memory_contexts and gather_views to exercise those code paths, and added rudimentary tests for the memory contexts. If you want the views to be more thoroughly tested, I'll have to add some fairly verbose test data to the bundled XML / JSON files.